lib/models/actor.dart

```diff
@@ -8,10 +8,12 @@ enum ActorType { npc, player }
 class Actor {
   final String id;
   ActorType type;
+  String avatarUrl;
 
   Actor(
     this.id, {
     this.type = ActorType.npc,
+    this.avatarUrl = '',
```

Comment:
Is an empty string really the best default? It might prevent some compile errors, but trying to load a picture from `''` will surely cause an error in run-time. Perhaps it should be nullable?

Reply:
In other words, the compiler will complain if you forget to check for `null`, but not so if you forget to check for `.isEmpty`

Reply:
It was nullable initially. But then I faced problem, that when user types and then clears the URL input fields – the field becomes empty strings again. To solve this I had to add into the editor `actor.avatarUrl = s.isEmpty ? null : s.trim()`. And then to add non-null checks into the indexing function too (for `coverUrl`). This solution feels both simpler and more elegant.